### PR TITLE
My hot fix

### PR DESCRIPTION
**Pull request by drets**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/templates/NewTestTemplate.java [hunk 1]
```diff
@@ -49,15 +49,17 @@ public void start(Method method, Object[] data) {
...
+      if (onDriver.length() > 0 & !onDriver.equalsIgnoreCase(Configuration.getBrowser())) {
+        throw new SkipException(
+            "The test can not be run on driver " + Configuration
+                .getBrowser() + ". The test is restricted to driver " + onDriver);
+      }
     }
 
+    startBrowser();
+
     if (method.isAnnotationPresent(Execute.class)) {
```
Comment (bognix) [inline src/test/java/com/wikia/webdriver/common/templates/NewTestTemplate.java]: consider combine those two ifs:
https://github.com/Wikia/selenium-tests/pull/847/files#diff-d7d0f5104feb18b29563c05b2d1f0e7cR52
and
https://github.com/Wikia/selenium-tests/pull/847/files#diff-d7d0f5104feb18b29563c05b2d1f0e7cR63

**Comment by justnpT** [comment]
lgtm, I will test this solution and merge if all is fine

File: src/test/java/com/wikia/webdriver/common/templates/NewTestTemplate.java (hunk 1 above)
Comment (drets) [inline src/test/java/com/wikia/webdriver/common/templates/NewTestTemplate.java]: @bognix thanks! 
We don't need start a browser if we run the test with inappropriate driver and we shouldn't load a first page if browser is not started. I don't see the way how to combine it, u?

Comment (bognix) [inline src/test/java/com/wikia/webdriver/common/templates/NewTestTemplate.java]: true :) miss that one on github :)

**Comment by bognix** [comment]
LGTM

**Comment by justnpT** [comment]
@drets  Thank you for the fix

**Comment by Archbold** [comment]
yep thanks :)

**Comment by drets** [comment]
you are welcome :smile:

**Pull request merged**